Sources/Rosetta/PlayMode/Tasks/SimpleTasks/DiscoverTask.cpp

```diff
@@ -356,6 +356,30 @@ std::vector<Card*> DiscoverTask::Discover(Game* game, Player* player,
                 }
             }
             break;
+        case DiscoverType::FILTER_STACK:
```

Comment:
```suggestion
    JANDICE_BAROV,
```

Reply:
Why don't you annotate the way this code works to help you understand?

Reply:
I changes DiscoverType::FILTER_STACK to DiscoverType::JANDICE_BAROV, 
so DiscoverType::JANDICE_BAROV doesn't need a process for removing duplicates.
Finally, I don't add annotation.

And I tried to add ChoiceAction::JANDICE_BAROV (because it already loses generality),
but I thought writing a specific id(SCH_351a, b) in Choose.cpp is a little bad design.
Also, I cannot get a unchosen card id in Choose.cpp.
So I didn't change it and use ChoiceAction::STACK continuously.

If you have any comment, notice me. Thanks.

Reply:
Alright. I agree. 👍